pkgs/development/python-modules/python-rapidjson/default.nix

```diff
@@ -1,37 +1,52 @@
 { lib
 , buildPythonPackage
+, fetchFromGitHub
 , fetchPypi
 , pythonOlder
-, pytest
+, rapidjson
+, pytestCheckHook
 , pytz
 , glibcLocales
 }:
 
-buildPythonPackage rec {
+let
+  rapidjson' = rapidjson.overrideAttrs (old: {
+    version = "unstable-2022-05-24";
+    src = fetchFromGitHub {
+      owner = "Tencent";
+      repo = "rapidjson";
+      rev = "232389d4f1012dddec4ef84861face2d2ba85709";
+      hash = "sha256-RLvDcInUa8E8DRA4U/oXEE8+TZ0SDXXDU/oWvpfDWjw=";
+    };
+    patches = lib.tail old.patches;
```

Comment:
This may cause issues in the future if rapidjson is changed without updating this